src/App.tsx

```diff
@@ -1,7 +1,37 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string
+};
+
+export class App extends React.Component<{}, State> {
+  state = {
+    pressedKey: 'none',
```

Comment:
none ?

Reply:
i just thought that no difference what default value, just needed to be string))